test/test.hs

```diff
@@ -562,7 +577,9 @@ tests = testGroup "hevm"
         a = BufLength (ConcreteBuf "ab")
         simp = Expr.simplify a
       assertEqualM "Must be simplified down to a Lit" simp (Lit 2)
-    , test "CopySlice-overflow" $ do
+    -- This test no longer works or needed, due to ReadWord simplification. Notice that it's writing
```

Comment:
Wait, so does this fail now or is it simply no longer needed?
I think the original purpose of the test was to check this weird overflow behaviour when the copy slice actually wraps back to start reading from the beginning again.
But that is such a can of worms and the SMT semantics seems to be different than EVM semantics.

Maybe we should just remove the test completely?

Reply:
Good point! Yeah, removing!